content/sensu-core/2.0/reference/silencing.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Silencing"
+description: "The silencing reference guide."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Silencing
+
+## How does silencing work?
+Silencing entries are created on an ad-hoc basis via `sensuctl`. When silencing entries are successfully created, they are assigned an `ID` in the format `$SUBSCRIPTION:$CHECK`, where `$SUBSCRIPTION` is the name of a Sensu agent subscription and `$CHECK` is the name of a Sensu check. Silencing entries can be used to silence checks on specific agents by by taking advantage of per-agent subscriptions, e.g. `agent:$AGENT_NAME`.
+
+These silencing entries are persisted in the Sensu data store. When the Sensu server processes subsequent check results, matching silencing entries are retrieved from the store. If one or more matching entries exist, the event is updated with a list of silenced entry ids. The presence of silencing entries indicates that the event is silenced.
```

Comment:
We should probably indicate what happen when an event is silenced. For example, in 1.x docs, we have something like this:

> When the check name and/or subscription described in a silencing entry match an event, the handler will not be executed and an informational message will be logged.

Reply:
Yes, let me add that, it should be the same behavior in 2.x. I'll double check that we do logging in that case.